wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/DebugActions/DeveloperDebugActionsViewModel.swift

```diff
@@ -159,4 +167,37 @@ final class DeveloperDebugActionsViewModel: ObservableObject {
         }
     }
 
+    // MARK: Delete domains
+
+    private func deleteDomains() {
+        guard let syncContext = userSession?.syncContext else {
+            print("failed to delete domains: no sync context")
+            return
+        }
+
+        syncContext.perform {
+            do {
+                print("deleted domains of users...")
+                let users = try syncContext.fetch(NSFetchRequest<ZMUser>(entityName: ZMUser.entityName()))
+
+                for user in users where !user.isSelfUser {
+                    user.domain = nil
+                }
+
+                print("deleted domains of conversations...")
+                let conversations = try syncContext.fetch(NSFetchRequest<ZMConversation>(entityName: ZMConversation.entityName()))
+
+                for conversation in conversations where conversation.conversationType.isOne(of: .oneOnOne, .group) {
+                    conversation.domain = nil
+                }
+
+                try syncContext.save()
+                print("successfully deleted domains")
+
+            } catch {
+                print("failed to delete domains: \(error.localizedDescription)")
```

Comment:
can we use a WireLogger might be useful to check on DD

Reply:
done